### DISCUSSION TITLE
Can the task of making ESM modules be broken up?

### DISCUSSION THREAD

**Discussion by bogdanbiv** [cytoscape/cytoscape.js#2835] [General]
Can the task of making ESM modules be broken up (https://github.com/cytoscape/cytoscape.js/issues/2019)?
Does there seem to be any good starting point? Would "remove internal polyfills" be a good fit?

**Comment by maxkfranz** [comment]
That would be a good first step to have slimmer ESM modules (exposed by package.json).
